Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Linker/LinkerPleaseInclude.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Specialized;
+using System.Windows.Input;
+using Android.App;
+using Android.Views;
+using Android.Widget;
+
+namespace MvvmSeed.Android.Linker
+{
+    /// <summary>
+    /// This file is required to avoid android Linker wiping references to code not detected during static analysis.
+    /// (We could disable linking, but I rather not, no linking means x3 or x4 size for builds)
+    /// </summary>
+    public class LinkerPleaseInclude
```

Comment:
It's hard to know which classes should be added and why 😄 

Most of the references added now are needed because MvvmCross' binding mechanisms works through reflection and a static code analysis won't catch those.

I'm not sure I can be more _precise_ than 'code not detected during static analysis'. I think this file will grow once we start seeing `MissingMethodExceptions` on release builds, not something we can easily anticipate. But it's not something that will happen often and most frameworks indicate if they need extra linker configuration for release builds.